### DISCUSSION TITLE
How to manage storage and logs?

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#7928] [Unsupported Versions] [answered]
Being new to Security Onion, I'm not sure where to start finding info on how to manage the space on disk for logs. Grafana shows that Disk Usage / nsm is at 90% after about a week. Since this is a lab, I don't need to store more than that. Is there a way to configure it to clear or purge after so many days? Is it automatic when it gets full to a point?

**Comment by Acewiza** [comment]
90% sounds like a good place to be, if it stays there.

4Tb has been hovering at around 75% full here for a few weeks now after the adjustment talked about in this thread:

[https://github.com/Security-Onion-Solutions/securityonion/discussions/6567]

That was just guesswork on my part, and I am certainly no expert on the various different file system types.

**Reply by ghost** [reply]
OK, but it there somewhere in the docs that explains how the logs are managed? I'm not seeing it. Will there be a situation where the logs get full and it stops working like when MSSQL Logs fill up?

**Reply by Acewiza** [reply]
Dunno, but sounds more like a "cross that bridge when I come to it" sort of thing, to me.

**Reply by ghost** [reply]
Can you point me to some ELI5 type explanations? Or where in the docs this is detailed? I'm not finding it. Thanks

**Comment by dougburks** [comment, marked as answer]
The two biggest users of disk space would be stenographer and Elasticsearch. You can read more about how they control their disk usage at:
https://docs.securityonion.net/en/2.3/stenographer.html#output
https://docs.securityonion.net/en/2.3/elasticsearch.html#deleting-indices

**Reply by ghost** [reply]
Thank you very much. I knew it was in there, I just didn't know where to look.
